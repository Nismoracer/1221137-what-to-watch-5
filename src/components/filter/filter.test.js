import React from "react";
import renderer from "react-test-renderer";
import Filter from "./filter";

const filterOne = () => {
  return (
    <li key="1" className="catalog__genres-item  catalog__genres-item--active">
      <a href="#" className="catalog__genres-link">Drama</a>
    </li>
  );
};
const filterTwo = () => {
  return (
    <li key="2" className="catalog__genres-item">
      <a href="#" className="catalog__genres-link">Comedy</a>
    </li>
  );
};

it(`Should render Filter correctly`, () => {
  const tree = renderer
    .create(
        <Filter
          filtersElement={[filterOne(), filterTwo()]}
          onFilterClick={()=>{}}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
