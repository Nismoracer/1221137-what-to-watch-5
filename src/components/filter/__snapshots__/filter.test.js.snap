// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Filter correctly 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item  catalog__genres-item--active"
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Drama
    </a>
  </li>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Comedy
    </a>
  </li>
</ul>
`;
